src/__tests__/migrate.js

```diff
@@ -297,13 +394,13 @@ test("hash check failure", (t) => {
   }
 
   const promise = createDb(databaseName, dbConfig)
-    .then(() => migrate(dbConfig, "src/__tests__/hash-check/first-run"))
-    .then(() => migrate(dbConfig, "src/__tests__/hash-check/second-run"))
+    .then(() => migrate(dbConfig, "src/__tests__/fixtures/hash-check/first-run"))
+    .then(() => migrate(dbConfig, "src/__tests__/fixtures/hash-check/second-run"))
 
   return t.throws(promise)
     .then((err) => {
       t.regex(err.message, /Hashes don't match/)
-      t.regex(err.message, /1_migration/, "Should name the problem file")
+      t.regex(err.message, /migration/, "Should name the problem file")
```

Comment:
In fact, I reckon this should stay as the full file name including the integer ID, rather than just the 'name' part of the file name. This will make it easier to find in the list of files.